source/troubleshooting.txt

```diff
@@ -0,0 +1,124 @@
+:noprevnext:
+
+.. _atlas-cli-troubleshooting:
+
+===================
+Troubleshoot Errors
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Command Errors
+--------------
+
+Error: missing credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure that you either:
+
+- Run ``atlas config init`` and add your API keys to your 
+  :ref:`profile <config-toml-location>` or add your API keys to your
+  :ref:`environment variables <atlas-cli-env-vars>`. If you created a
+  profile with a custom name or are using more than one profile,
+  specify the correct profile with the ``--profile`` flag.
+- Run ``atlas auth login`` to authenticate using your |service| login
+  credentials and an authentication token.
+
+To learn more, see :ref:`connect-atlas-cli`.
+
+atlas: command not found
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``atlas`` executable might be in a directory which is not in your
+path. You can either add the directory to your path, move the executable
```

Comment:
should this be 
```suggestion
$PATH. You can either add the directory to your path, move the executable
```
to make it clear what path?

Reply:
Sure, will change it. Thanks!